rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/docker/DockerImageNameFactory.groovy

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2018 Netflix, Inc.
```

Comment:
Could verify this by viewing history for each file, but fairly certain the copyright for all of these *Util classes should be Schibsted.

Reply:
Perhaps at some point copyrights could be relegated to "Spinnaker Contributors" or something like that. Believe there are other projects that do something similar.

Reply:
I'll just remove license updates here. It was just an attempt to get nagging warnings out of the build anyway.